alma/alma-woocommerce-gateway
# Feature/ecom 1402 wc blocks pnx and credit blocks are displayed

## Reason for change

Display correctly with design the pnx block.

[Linear task]https://linear.app/almapay/issue/ECOM-1402/%5B%F0%9F%92%85-wc-blocks%5D-pnx-and-credit-blocks-are-displayed)

### How to test

On woocommerce 8-5-1 or 8-6-0, on the checkout page, you should be able to see the new block.
![image](https://github.com/alma/alma-woocommerce-gateway/assets/112871182/2a225676-bc70-48b0-843a-d2a1c58aa4f6)
